### PR TITLE
feat(react-spring): Created blog post for react-spring hooks

### PR DESCRIPTION
**Pull request by nmarini**

### CODE REVIEW

File: src/content/react-spring/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+repo: https://github.com/nmarini/animated
+---
+
+The [react-spring](https://www.react-spring.io/) animation library is a spring-physics based animation library. It gives you an array of tools to easily and elegantly create animations.
+
+This post is going to specifically focus on the react-spring’s Hook api.
+
+This library presents a more modern approach to animations - it’s physics based instead of relying on css transitions. So animation effects are more in line with real world physics.
+
+## Why React Springs
```
Comment (trybick) [inline src/content/react-spring/index.md]: should this be Spring instead of Springs?

File: src/content/react-spring/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+react-spring is easy to use and takes a lot less code to create the desired animations. Using react-spring is also a very easy transition given React already utilizes hooks. The react-spring hooks are also very intuitive, allowing a developer to right less code but still achieve a very natural and elegant animation.
```
Comment (trybick) [inline src/content/react-spring/index.md]: spelling - "right less code"

File: src/content/react-spring/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+title: 'Introduction to React-Spring Hooks'
+date: '2020-11-19'
+author: 'Nick Marini'
+author_site: https://github.com/nmarini
+repo: https://github.com/nmarini/animated
+---
+
+The [react-spring](https://www.react-spring.io/) animation library is a spring-physics based animation library. It gives you an array of tools to easily and elegantly create animations.
+
+This post is going to specifically focus on the react-spring’s Hook api.
```
Comment (trybick) [inline src/content/react-spring/index.md]: throughout the article, should we capitalize API?

**Review by trybick — COMMENTED** [review]
@nmarini Hey Nic! Sorry for the delay on this. Looks good other than a few small comments.

File: src/content/react-spring/index.md (hunk 2 above)
Comment (nmarini) [reply src/content/react-spring/index.md]: I'm the worst speller haha 🤦

Comment (trybick) [reply src/content/react-spring/index.md]: hahaha, happens to all of us

**Review by trybick — APPROVED** [review]
LGTM

**Pull request merged**